rasterio/rio/helpers.py

```diff
@@ -83,7 +83,6 @@ def resolve_inout(input=None, output=None, files=None, force_overwrite=False):
     :param:`force_overwrite` is `True`.
     """
     resolved_output = output or (files[-1] if files else None)
-    force_overwrite = output is not None or force_overwrite
     if not force_overwrite and resolved_output and os.path.exists(
             resolved_output):
         raise FileOverwriteError(
```

Comment:
@sgillies the error message here needs to be updated to drop the mention of `--output` option.